### PR TITLE
Add support for linux_ppc64le and linux_aarch64

### PR DESCRIPTION
**Pull request by count0**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by stuarteberg** [comment]
The new platforms won't be built unless we re-render the feedstock, which will add the appropriate yaml files in `.ci_support`.

So here goes...

@conda-forge-admin, please rerender

**Comment by stuarteberg** [comment]
Hmm... Rerendering added the new configuration files for `linux-aarch64` and `linux-ppc64le`, but it didn't start new checks for those platforms -- it just restarted the checks that it was already running.

I wonder if closing this PR and re-opening it would properly reinitialize the CI and allow it to start new checks.

**Comment by jakirkham** [comment]
Also we should add this to the arch migration so the bot is aware of it ( https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/4260 )

**Comment by stuarteberg** [comment]
1. On Gitter, Isuru pointed out that re-rendering DID work.  These platforms are built by Travis-CI, not Azure.  I wasn't looking at the right checks.  My bad.  (Azure doesn't have native `aarch64` support, but Travis does, I guess.)

2. @jakirkham I opened https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/4261

**Comment by stuarteberg** [comment]
BTW, Isuru also pointed out that if Travis is misbehaving ("it's flaky"), then another option is to build on Azure via cross-compilation:

```yaml
build_platform:
  linux_ppc64le: linux_64
  linux_aarch64: linux_64
```

...but meanwhile, IIUC, a bot will start opening PRs on this repo and the others mentioned in the above-mentioned PR once it is merged.  If something goes wrong here, I'll probably wait to see how the bot's PR looks.

**Comment by jakirkham** [comment]
Here's the migrator PR ( https://github.com/conda-forge/cairomm-feedstock/pull/30 ) FWIW. If it is merged or closed, the bot will treat the feedstock as migrated. So if we go with this PR, would make sure to close the bot PR

**Comment by isuruf** [comment]
Let's do cross compilation. I don't want to use travis unless it's absolutely necessary.

**Comment by jakirkham** [comment]
Could you please propose those changes in the diff then?

File: conda-forge.yml [hunk 1]
```diff
@@ -7,3 +7,6 @@ github:
   tooling_branch_name: main
 conda_build:
   pkg_format: '2'
+provider:
+  linux_ppc64le: default
+  linux_aarch64: default
```
Comment (stuarteberg) [inline conda-forge.yml]:
```suggestion
  linux_ppc64le: linux_64
  linux_aarch64: linux_64
```

**Comment by stuarteberg** [comment]
>Could you please propose those changes in the diff then?

Done.

**Comment by stuarteberg** [comment]
@count0 Can you apply the suggested change, since this is your PR?

**Comment by stuarteberg** [comment]
@count0 You also need to increment the build number in `meta.yaml`:

```yaml
build:
  number: 1
  run_exports:
    - cairomm-{{ abi_version }} >={{ version }}
```

**Comment by jakirkham** [comment]
@conda-forge-admin, please re-render

**Comment by jakirkham** [comment]
Re-rendering is needed for the `conda-forge.yml` change. However it looks like the new builds were removed entirely. Usually this is due to other changes being needed in the recipe

**Comment by stuarteberg** [comment]
Looks like a random network error. It failed to download the source tarball.

@conda-forge-admin, please restart ci

**Comment by stuarteberg** [comment]
Please close.  Superseded by #31.

(This PR had a bug, due to a mistaken code suggestion from me.  My bad.)

**Pull request closed without merging**